### PR TITLE
[Fix] Update the maximum version of mmcv

### PR DESCRIPTION
**Pull request by qipengh**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Fix maximum version check of mmcv . Issue [#2243](https://github.com/open-mmlab/mmdetection3d/issues/2243)

## Modification

modify mmdet3d/__init__.py with mmcv_maximum_version from '1.7.1' to '1.8.0'

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by qipengh** [comment]
Issue [#2243](https://github.com/open-mmlab/mmdetection3d/issues/2243)

File: mmdet3d/__init__.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def digit_version(version_str):
 
 
 mmcv_minimum_version = '1.5.2'
-mmcv_maximum_version = '1.7.0'
+mmcv_maximum_version = '1.8.0'
```
Comment (JingweiZhang12) [inline mmdet3d/__init__.py:22]: Please modify the version of MMCV at https://github.com/open-mmlab/mmdetection3d/blob/master/docs/en/faq.md and https://github.com/open-mmlab/mmdetection3d/blob/master/docs/zh_cn/faq.md

Comment (qipengh) [reply mmdet3d/__init__.py:22]: done.

Comment (qipengh) [reply mmdet3d/__init__.py]: why not this change sync to master branch? I have a little doubt.

Comment (ZwwWayne) [reply mmdet3d/__init__.py]: we usually merge pr into dev first, then merge dev into master.

**Comment by ZwwWayne** [comment]
Both the document and __init__.py should be modified.

File: mmdet3d/__init__.py (hunk 1 above)
Comment (qipengh) [reply mmdet3d/__init__.py]: The dev branch has been modified to mmcv_maximum_version=1.8.0. So this PR is useless. Just need to synchronize dev to master later. thanks~

**Comment by OpenMMLab-Assistant001** [comment]
Hi @qipengh  ！We are grateful for your efforts in helping improve this open-source project during your personal time.

Welcome to join OpenMMLab Special Interest Group (SIG) private channel on **Discord**, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA
If you have a WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "**mmsig + Github ID**" as a remark when adding friends：）

Thank you again for your contribution❤

**Pull request closed without merging**